secretless-k8s-tutorial/index.json

```diff
@@ -0,0 +1,93 @@
+{
+  "title": "CyberArk Secretless Broker on Kubernetes",
+  "icon": "fa-kubernetes",
+  "description": "Tutorial: Running CyberArk Secretless Broker on Kubernetes",
+  "difficulty": "Beginner",
+  "time": "30 minutes",
+  "details": {
+    "steps": [
+      {
+        "title": "Step 1",
```

Comment:
Nit: I wonder if it would be better to use the format of 

"title": "Step 1: Name of Step"

As is, it's a little ambiguous knowing what each step is without reading the file itself